controller/Previewer.php

```diff
@@ -137,15 +156,15 @@ public function init()
         try {
             $this->validateCsrf();
 
-            $serviceCallId = $this->getRequestParameter('serviceCallId');
+            $requestParams = $this->getPsrRequest()->getQueryParams();
+            $serviceCallId = $requestParams['serviceCallId'];
 
             $response = [
-                'success' => $serviceCallId == 'previewer',
+                'success' => $serviceCallId === 'previewer',
                 'itemIdentifier' => null,
                 'itemData' => null
             ];
-
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
             $response = $this->getErrorResponse($e);
             $code = $this->getErrorCode($e);
```

Comment:
it might be that we need a default 200 declaration because the refactored version does not return 200 like previously